packages/core/src/llm/services/test-utils.integration.ts

```diff
@@ -74,7 +74,11 @@ export const TestConfigs = {
             },
             sessions: {
                 maxSessions: 10,
-                sessionTTL: 100, // 100ms for fast testing
+                sessionTTL: 60000, // 60s for tests
+            },
+            logger: {
+                level: 'debug',
```

Comment:
keep it info, we can override with dexto_log_level now right? similar for other test utils here